typescript.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "commitMessageAction": "update",
```

Comment:
nit: since in `packageRules` section we're matching to dependabot, should we maybe use `bump` here as well?

Reply:
Done in fa50655.